src/ex4/server/routes/api.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const router = express.Router();
+const itemManager = require("../services/item_manager");
+
+router.get("/items", (_, res) => {
+  console.log("api - getItems:", itemManager.getItems());
+  res.send(itemManager.getItems());
```

Comment:
i guess you know this function doesn't work, and for a few reasons, but the main one is the fact that you're not sending a json object but something else...